lib/internal/process/warning.js

```diff
@@ -1,9 +1,74 @@
 'use strict';
 
+const config = process.binding('config');
 const prefix = `(${process.release.name}:${process.pid}) `;
 
 exports.setup = setupProcessWarnings;
 
+var fs;
+var fd;
+var acquiringFd = false;
+function nop() {}
+
+function lazyFs() {
+  if (!fs)
+    fs = require('fs');
```

Comment:
Depends on what you mean. This uses async fs methods so in that sense no, but since `--trace-sync-io` uses process.emitWarning, then using `--redirect-warnings` does redirect `--trace-sync-io` output to the warning file.